proto/viam/app/cloudslam/v1/cloud_slam.proto

```diff
@@ -74,14 +77,33 @@ message ListMappingSessionsResponse {
 
 message StopMappingSessionRequest {
   string session_id = 1;
-  bool save_map = 2;
 }
 
 message StopMappingSessionResponse {
   string package_id = 1;
   string version = 2;
 }
 
+// GetMappingSessionMetadataByID
+
+message GetMappingSessionMetadataByIDRequest {
+  string session_id = 1;
+}
+
+message GetMappingSessionMetadataByIDResponse {
+  MappingMetadata session_metadata = 1;
+}
+
+// UpdateMappingSessionMetadataByID
+
+message UpdateMappingSessionMetadataByIDRequest {
+  string session_id = 1;
+  string end_status = 2;
```

Comment:
[nit] add a comment for end status, similar to the one in MappingMetadata

Reply:
done!